src/solid/nossr.ts

```diff
@@ -0,0 +1,25 @@
+import { Event, Store } from "effector";
+import { useStoreBase, useStoreMapBase } from "./lib/base";
+import { Accessor } from "solid-js";
+
+export function useEvent<T>(event: Event<T>): (payload: T) => T {
+  return event
+}
+
+export function useStore<State>(store: Store<State>): Accessor<State> {
+  return useStoreBase(store)
+}
+
+export function useStoreMap<State, Result, Keys extends ReadonlyArray<any>>(
+  configOrStore:
+    | {
+    store: Store<State>
+    keys: Keys
+    fn(state: State, keys: Keys): Result
+    updateFilter?: (update: Result, current: Result) => boolean
+  }
+    | Store<State>,
+  separateFn?: (state: State, keys: Keys) => Result,
+): Accessor<Result> {
+  return useStoreMapBase([configOrStore, separateFn])
+}
```

Comment:
What about to remove `useEvent` and `useStore` in favor of `useUnit`?

https://github.com/effector/effector/issues/680

@zerobias what do you think? What if we can do not add methods in the new bindings.

> But we still need `useStoreMap`

Reply:
I thought a little more and now i think it is better to start with `useUnit` in solid right away 🙂 

It is just one api instead of two, supports built-in batching, and easily usable just like old apis

Since bindings for solid are not yet bothered by being backward-compatible, i think, it is better to favor simpler api

Also, it is still will be easily rewritable from react like this:

From:
```ts
const a = useStore($a)
const b = useEvent(b)
```
to:
```ts
const a = useUnit($a)
const b = useUnit(b)
```
or:
```ts
const {a, b} = useUnit({a: $a, b})
```

Reply:
Done!